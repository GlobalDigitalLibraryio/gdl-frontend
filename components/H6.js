// @flow
/**
 * Part of GDL gdl-frontend.
 * Copyright (C) 2017 GDL
 *
 * See LICENSE
 */

import styled from 'react-emotion';
import media from '../style/media';

const H6 = styled.h6`
  margin-top: 0;
  margin-bottom: 0;
  text-transform: uppercase;
  font-weight: bold;
  font-size: 12px;
  line-height: 16px;
  ${media.tablet`
    font-size: 14px;
    line-height: 18px;
  `};
`;

export default H6;
