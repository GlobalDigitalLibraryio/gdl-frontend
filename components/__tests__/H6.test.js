// @flow
/**
 * Part of GDL gdl-frontend.
 * Copyright (C) 2017 GDL
 *
 * See LICENSE
 */

import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { createSerializer } from 'jest-emotion';
import * as emotion from 'emotion';
import H6 from '../H6';

expect.addSnapshotSerializer(createSerializer(emotion));

test('Renders', () => {
  const tree = shallow(<H6>Header 6</H6>);

  expect(toJson(tree)).toMatchSnapshot();
});
