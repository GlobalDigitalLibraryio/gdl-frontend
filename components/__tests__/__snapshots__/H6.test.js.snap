// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
.c0 {
  margin-top: 0;
  margin-bottom: 0;
  text-transform: uppercase;
  font-weight: bold;
  font-size: 10px;
  line-height: 16px;
}

@media (min-width:40em) {
  .c0 {
    font-size: 12px;
    line-height: 18px;
  }
}

<h6
  className="c0"
>
  Header 6
</h6>
`;
